mne/stats/regression.py

```diff
@@ -134,3 +142,197 @@ def _fit_lm(data, design_matrix, names):
         mlog10_p_val[predictor] = -np.log10(p_val[predictor])
 
     return beta, stderr, t_val, p_val, mlog10_p_val
+
```

Comment:
And the original rERP idea is to forget about the fitted model and instead do e.g. inference on the resulting coefs across subjects - so you just treat your coefs like single-subject across-trial mean values.